### PR TITLE
Test against all HHVM LTS releases

### PR DESCRIPTION
**Pull request by WillSewell**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -55,7 +55,12 @@ matrix:
     # Test PHP 7.2 without LibSodium since it adds support natively.
     - php: 
       - 7.2
+      # Test HHVM LTS releases: http://docs.hhvm.com/hhvm/installation/release-schedule#supported-releases
       - hhvm-3.24
+      - hhvm-3.27
+      - hhvm-3.29
```
Comment (damdo) [inline .travis.yml]: It looks like `hhvm-3.29` is not LTS, do we want to test it anyway?
`hhvm-4.0` is not LTS but is the latest stable release. If we want to always test the latest one does it make sense to use `hhvm` without the version specified (it fetches the latest stable) instead?

Comment (WillSewell) [reply .travis.yml]: 3.29 and 4.0 are according to this: http://docs.hhvm.com/hhvm/installation/release-schedule#supported-releases

The main reason I'd rather not use `latest` is that hhvm no longer officially supports PHP (as of 4.0). So I don't think we should try to continue to support it at that point (we'd probably need to create a new library if anything).

Comment (damdo) [reply .travis.yml]: I'm referring to the _Supported Releases_ table in this link http://docs.hhvm.com/hhvm/installation/release-schedule#supported-releases under the LTS column it says that 4.0 and 3.29 are not LTS, or am I missing something?

Oh I see the point for `4.0`, 👍

Comment (WillSewell) [reply .travis.yml]: Oh sorry! I didn't see the "LTS" column. I thought it was just a list of all supported! Yes, I think you're right then.

I'll leave testing against 4.0 for now. I was more just curious to see whether the library would still be compatible.

**Pull request merged**